### PR TITLE
slof_memory: Add slof_memory case for testing SLOF

### PR DESCRIPTION
**Pull request by YongxueHong**
slof_memory case is that verify SLOF info
with large maxmem.

ID:1645055

Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

**Comment by YongxueHong** [comment]
Depends on #1531

**Comment by wangxianxian** [comment]
qemu.slof_memory.with_maxmem_mem: PASS

Acked by xianwang@redhat.com

File: qemu/tests/slof_memory.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      c. ping external host ip successfully.
+
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    start_pos = 0
+    for mem in params['maxmem_mem_list'].split():
+        params['maxmem_mem'] = mem
+        params['start_vm'] = 'yes'
```
Comment (hereischen) [inline qemu/tests/slof_memory.py]: I think you can put `start_vm = yes` in the cfg

**Comment by YongxueHong** [comment]
@lolyu  help me to review, thanks.

File: qemu/tests/slof_memory.py (hunk 1 above)
Comment (YongxueHong) [reply qemu/tests/slof_memory.py]: Thanks for your comments,   have updated.

**Pull request merged**